core/server/src/main/webapp/general.jsp

```diff
@@ -114,6 +114,38 @@
       </div>
     </div>
   </div>
+  <!--  show inconsistent path  -->
+  <% if ((Integer) request.getAttribute("inconsistentPaths") != 0) { %>
+  <div class="row-fluid">
+    <div class="accordion span14" id="accordion4">
+      <div class="accordion-group">
+        <div class="accordion-heading">
+          <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion4" href="#data4">
+            <h4>Inconsistent Files</h4>
```

Comment:
Could you clarify this is only for the startup check and may not be up to date? Also that the user can get the updated status by restarting Alluxio?